test/avalanche_test.exs

```diff
@@ -367,4 +489,20 @@ defmodule AvalancheTest do
                Avalanche.status(statement_handle, [], request_options)
     end
   end
+
+  defp expect_telemetry_mock_start do
+    expect(TelemetryDispatchBehaviourMock, :execute, fn [:avalanche, :query, :start],
+                                                        %{system_time: _},
+                                                        %{params: _, query: _} ->
+      :ok
+    end)
+  end
+
+  defp expect_telemetry_mock_stop do
+    expect(TelemetryDispatchBehaviourMock, :execute, fn [:avalanche, :query, :stop],
+                                                        %{duration: _},
+                                                        %{params: _, query: _} ->
```

Comment:
Can we test stop with an `error` in the metadata? Not an exception.

Reply:
dca7af55b2d13c6b30ed9af48714cdb25b537c95